plugins/modules/suma.py

```diff
@@ -256,7 +256,7 @@ def find_sp_version(file):
     return:
        sp_version   value found or None
     """
-    sp_version = None
+    sp_version = ""
```

Comment:
The line 270 has a check if the sp_version is None or ... 
Now that the sp_version = "",  the sp_version may not be updated in the first time of the iteration after a match.

Reply:
The first time that there is a match, it will be something in the version variable like "7200-05-04-02114". When that is compared to sp_version the first time, sp_version is "", and the condition 'version > sp_version' will be true. I think that the first condition 'sp_version is None' is not necessary.